packages/client/src/workspace/apps.ts

```diff
@@ -1,20 +1,32 @@
 import { App } from "@openfin/workspace"
 
-import { BASE_URL } from "./consts"
-import { manifestUrls } from "./utils"
+import { BASE_URL, manifestUrls } from "./constants"
+import { getCurrentUser, USER_TRADER } from "./user"
 
-export async function getApps(): Promise<App[]> {
-  return [
-    reactiveTraderFx,
-    reactiveTraderCredit,
-    reactiveAnalytics,
-    reactiveWorkspace,
-    limitCheckerView,
-    reactiveTraderFxLiveRatesView,
-    reactiveTraderFxTradesView,
-    reactiveTraderFxAnalyticsView,
-    reactiveAnalyticsView,
-  ]
+export function getApps(): App[] {
+  return getCurrentUser() === USER_TRADER
+    ? [reactiveTraderFx, reactiveTraderCredit, reactiveAnalytics]
+    : [reactiveAnalytics, limitChecker]
```

Comment:
This might just be me not understanding limit checker properly, but wouldn't a user want access to both limit checker and FX?

Reply:
no, this is where I fell down earlier creating a workspace with Live Trades and LC .. the whole point of the Limit Checker is that it is being used to control a Trader, so Trader should not have access (makes the interop difficult, I know!) .. anyway, we can demo by opening LC as Operator and trading as Trader .. smoke and mirrors but it's always going to be difficult demoing as a single user (ref also Credit demo, with pretend "sell side" popup ..)